dbms/src/Interpreters/ExternalLoaderPresetConfigRepository.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <Core/Types.h>
+#include <unordered_map>
+#include <Interpreters/IExternalLoaderConfigRepository.h>
+#include <Poco/Timestamp.h>
+
+
+namespace DB
+{
+/// A config repository filled with preset loadables used by ExternalLoader.
```

Comment:
This comment doesn't help.

Reply:
I think I'll rename this class to something like `ExternalLoaderTempConfigRepository` and write a decent comment in a separate PR.